client/src/views/Search.tsx

```diff
@@ -0,0 +1,119 @@
+import Select from '@mui/material/Select';
+import InputBase from '@mui/material/InputBase';
+import SearchIcon from '@mui/icons-material/Search';
+import Divider from '@mui/material/Divider';
+import MenuItem from '@mui/material/MenuItem';
+import { SelectChangeEvent } from '@mui/material';
+// import NativeSelect from '@mui/material/NativeSelect';
+import Avatar from '@mui/material/Avatar';
+const useStyles = makeStyles((theme: Theme) => ({
+  input: {
+    marginLeft: theme.spacing(3),
+    flex: 10,
+  },
+  iconButton: {
+    padding: 10,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    backgroundColor: 'black',
+    borderRadius: '25px',
+    '&:hover': {
+      backgroundColor: 'inheret',
+      borderRadius: '10px',
+    },
+  },
+  searchBar: {
+    fontFamily: 'DM Sans',
+    display: 'flex',
+    position: 'relative',
+    flexDirection: 'row',
+    background: 'white',
+    borderRadius: '40px',
+    width: '100%',
+  },
+  formControl: {
+    margin: theme.spacing(2),
+    minWidth: 90,
+    borderRadius: '40px',
+    borderColor: 'pink',
+    notched: 'false',
+  },
+}));
+function Search() {
+  const classes = useStyles();
+  const [selectedSearchCategory, setSelectedSearchCategory] = React.useState<string>('');
+  const [searchText, setSearchText] = React.useState<string>('');
+
+  const selectSearchCategory = (event: SelectChangeEvent<string>) => {
+    setSelectedSearchCategory(event.target.value as string);
+  };
+  return (
+    <div className={classes.searchBar}>
+      <FormControl variant="standard" sx={{ borderRadius: '20px' }}>
+        <Select
+          displayEmpty
+          value={selectedSearchCategory}
+          onChange={selectSearchCategory}
+          renderValue={(value: any) => value || 'Search All'}
+          MenuProps={{
+            anchorOrigin: {
+              vertical: 52,
+              horizontal: -28,
+            },
+            transformOrigin: {
+              vertical: 'top',
+              horizontal: 'left',
+            },
+            PaperProps: {
+              sx: { borderRadius: '20px', boxShadow: '0px -2px 1em grey' },
+            },
+          }}
+          sx={{
+            fontSize: '20px',
+            border: 0,
+            padding: 1.5,
+            paddingLeft: 3,
+            paddingRight: 5,
+          }}
+          disableUnderline
+        >
+          <MenuItem value="All">Search All</MenuItem>
+          <MenuItem value="Nonprofits">Search Nonprofits</MenuItem>
+          <MenuItem value="Needs">Search Needs</MenuItem>
+          <MenuItem value="Offers">Search Offers</MenuItem>
+          <MenuItem value="Volunteer">Volunteer Openings</MenuItem>
+        </Select>
+      </FormControl>
+      <Divider
+        sx={{ height: 62, marginLeft: 1, borderRightWidth: 3, bgcolor: 'grey' }}
+        orientation="vertical"
+      />
+      <InputBase
+        className={classes.input}
+        sx={{ notched: 'false' }}
```

Comment:
can we move all of the styling into `makeStyles` (preferred) or `sx`
i think it's more readable if they're all in one place